bms/bms_app/static/bms_app/css/create_project.css

```diff
@@ -118,28 +72,28 @@ label {
 }
 
 #user-count{
-	background-color: #81d4e4;
+	background-color: #D0D0D0;
 	border-radius: 4px;
 	margin-bottom: 10px;
 	padding: 5px 0;
 }
 
 #submit-data{
-	margin-right: 30px;
-	background-color:  #5093e1;
-	border: none;
-	color: #fff;
+	margin-right: 15px;
 }
 
 @media (max-width: 767px){
 	#user-role{
-		margin: 15px 0px;
+		margin: 15px 0px 15px 0px;
```

Comment:
Repeated mistake..... Read about CSS short hand properties and implement it as per use case.